### ISSUE TITLE
Missing kit-schema/locale directory with 1.2.0

### ISSUE THREAD

**Issue by pthorod** [microsoft/vscode-cmake-tools#874]
### Brief Issue Summary

Missing CMake kit schema locale / directory after insalling 1.2.0 as .vix package. 

### Expected:

1. Download and install newly released version of this extension as .vix package
2. Open vscode on cmake project folder with `.vscode/cmake-kits.json` file
3. vscode opens without warnings.

### Apparent Behavior:

3. vscode opens with warning from cmake-tools extension as below:

### CMake Tools Log
```
Request vscode/content failed with message: cannot open cmake-tools-schema://schemas/kits-schema.json. Detail: ENOENT: no such file or directory, stat 'c:\Users\preben\.vscode\extensions\ms-vscode.cmake-tools-1.2.0\dist\schema\en-us\schemas'
```

### Platform and Versions
- **Operating System**: Windows 10
- **CMake Version**: 3.15.3
- **VSCode Version**: 1.39.2 
- **CMake Tools Extension Version**: 1.2.0 (Downloaded from GitHub releases as vix)

### schema folders after installation:
```
 Directory of C:\Users\preben\.vscode\extensions\ms-vscode.cmake-tools-1.2.0\dist\schema

06-Nov-19  20:48    <DIR>          .
06-Nov-19  20:48    <DIR>          ..
06-Nov-19  20:48    <DIR>          cs
06-Nov-19  20:48    <DIR>          de
06-Nov-19  20:48    <DIR>          es
06-Nov-19  20:48    <DIR>          fr
06-Nov-19  20:48    <DIR>          it
06-Nov-19  20:48    <DIR>          ja
06-Nov-19  20:48    <DIR>          ko
06-Nov-19  20:48    <DIR>          pl
06-Nov-19  20:48    <DIR>          pt-br
06-Nov-19  20:48    <DIR>          ru
06-Nov-19  20:48    <DIR>          tr
06-Nov-19  20:48    <DIR>          zh-CN
06-Nov-19  20:48    <DIR>          zh-TW
               0 File(s)              0 bytes
              15 Dir(s)  180,836,823,040 bytes free
```

```
Directory of C:\Users\preben\.vscode\extensions\ms-vscode.cmake-tools-1.2.0\schemas

06-Nov-19  20:48    <DIR>          .
06-Nov-19  20:48    <DIR>          ..
06-Nov-19  20:48             2,597 kits-schema.json
06-Nov-19  20:48             3,500 variants-schema.json
               2 File(s)          6,097 bytes
               2 Dir(s)  180,833,914,880 bytes free
```

**Comment by bobbrow** [comment]
Thanks for reporting this.  We are working on a fix.

**Comment by pthorod** [comment]
My pleasure, let me know if you need more information.

**Comment by bobbrow** [comment]
This is fixed in 1.2.1.  Auto-update should be working now.  Please let us know if you run into any other issues.
